### PR TITLE
added CREDITS.md, updated README.md

### PR DESCRIPTION
**Pull request by pycoinfu**
Moved the credits to a separate file; removed the `Welcome to the Pygame Guide for Sneks!` line, as generally there shouldn't be anything above the logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-Welcome to the Pygame Guide for Sneks!
```
Comment (oddbookworm) [inline README.md:1]: Should we add this back in under the logo?

**Review by oddbookworm — APPROVED** [review]

**Review by oddbookworm — APPROVED** [review]

**Comment by oddbookworm** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,20 +1,11 @@
...
-
 <div align="center"><img src="./logo_files/logo.svg" width="325"/></div>
+Welcome to the Pygame Guide For Sneks!
```
Comment (Matiiss) [inline README.md:2]: Could this be centered under the logo?

**Review by Matiiss — APPROVED** [review]
LGTM

**Pull request merged**